src/rdf_graph/map_queries.py

```diff
@@ -322,9 +322,24 @@ def run(args_dict):
 
     # fit run-determined parameters
     if 'fit' in args_dict['run']:
-        con.set_train_times(1000)
-        con.set_nbatches(100)
-        con.set_alpha(0.001)
+        if args_dict['traintimes'] is None:
```

Comment:
This code is unnecessary; set the default as part of the argument (there is a `default` option in the `add_argument()` method. Not worth changing now, but please be aware of this for the future.